### ISSUE TITLE
Fix: Ratings Section

### ISSUE THREAD

**Issue by dani017** [collegescore/frontend#86]
Improve ratings section experience. Fix so things are no longer headings, still syled as is but should be p, and wrap in a div to make screen reader flow better. Currently screen readers read the existing section as "heading level 6, 3.9" then later "accessibility" after these changes will read as "Accessibility 3.9 out of 5"
```bash
<dl>
  <div
    class="MuiBox-root mui-dvxtzn"
    role="group"
    aria-label="Accessibility 3.9 out of 5"  <!-- ✅ what screen reader announces -->
  >
    <dd class="MuiBox-root mui-11n2qie" aria-hidden="true">  <!-- 👁️ visual only -->
      <svg aria-hidden="true" ...>...</svg>
      <p>3.9</p>
    </dd>
    <dt aria-hidden="true">Accessibility</dt>  <!-- 👁️ visual only -->
  </div>
</dl>
```
